tests/test-work.lua

```diff
@@ -20,6 +20,8 @@ return require('lib/tap')(function (test)
                 if (step % 100==0) then
                     p(string.format('run %d%%', math.floor(step*100/count)))
                 end
+            else
+              ctx = nil
```

Comment:
Same question here. Is this now necessary?

Reply:
Yes, need set ctx to nil.  because ctx -> callback -> ctx loop,  if ctx not nil,  ctx will not be gc.

Reply:
You're right, and this was always true (`__gc` didn't get called in this test in older versions of `luv` too without this `ctx = nil` change)